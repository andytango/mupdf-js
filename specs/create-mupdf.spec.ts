import { createMuPdf } from "../dist";

describe("createMuPdf", () => {
  it("should create a MuPdf instance", async () => {
    const mupdf = await createMuPdf();
    expect(mupdf).toBeDefined();
    expect(Object.getOwnPropertyNames(mupdf)).toMatchSnapshot();
  });
});
