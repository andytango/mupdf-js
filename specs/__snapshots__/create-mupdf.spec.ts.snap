// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`createMuPdf should create a MuPdf instance 1`] = `
[
  "load",
  "ready",
  "noExitRuntime",
  "noInitialRun",
  "locateFile",
  "onRuntimeInitialized",
  "documentOutline",
  "inspect",
  "HEAP8",
  "HEAP16",
  "HEAP32",
  "HEAPU8",
  "HEAPU16",
  "HEAPU32",
  "HEAPF32",
  "HEAPF64",
  "preloadedImages",
  "preloadedAudios",
  "___wasm_call_ctors",
  "_initContext",
  "_openDocument",
  "_freeDocument",
  "_countPages",
  "_drawPageAsHTML",
  "_drawPageAsSVG",
  "_drawPageAsPNG",
  "_pageWidth",
  "_pageHeight",
  "_pageLinks",
  "_documentTitle",
  "_loadOutline",
  "_freeOutline",
  "_outlineTitle",
  "_outlinePage",
  "_outlineDown",
  "_outlineNext",
  "_getPageText",
  "_malloc",
  "_searchPageText",
  "___errno_location",
  "__get_tzname",
  "__get_daylight",
  "__get_timezone",
  "_setThrew",
  "stackSave",
  "stackRestore",
  "stackAlloc",
  "dynCall_v",
  "dynCall_vi",
  "dynCall_vii",
  "dynCall_viii",
  "dynCall_viiii",
  "dynCall_viiiii",
  "dynCall_viiiiii",
  "dynCall_viiiiiii",
  "dynCall_viiiiiiii",
  "dynCall_viiiiiiiii",
  "dynCall_viiiiiiiiii",
  "dynCall_viiiiiiiiiii",
  "dynCall_viiiiiiiiiiiii",
  "dynCall_viiiiiiif",
  "dynCall_viiiiiiifi",
  "dynCall_viiiiiiifiiifffffiii",
  "dynCall_viiiiiifi",
  "dynCall_viiiiifii",
  "dynCall_viiiif",
  "dynCall_viiiifi",
  "dynCall_viiiiff",
  "dynCall_viiij",
  "dynCall_viiif",
  "dynCall_viiifffffiii",
  "dynCall_viiid",
  "dynCall_viij",
  "dynCall_viiji",
  "dynCall_viif",
  "dynCall_viifi",
  "dynCall_viiff",
  "dynCall_viiffii",
  "dynCall_viifff",
  "dynCall_viifffiiff",
  "dynCall_viiffff",
  "dynCall_viiffffff",
  "dynCall_viid",
  "dynCall_vij",
  "dynCall_ii",
  "dynCall_iii",
  "dynCall_iiii",
  "dynCall_iiiii",
  "dynCall_iiiiii",
  "dynCall_iiiiiii",
  "dynCall_iiiiiiii",
  "dynCall_iiiiiiiii",
  "dynCall_iiiiiiiiii",
  "dynCall_iiiiiiiiiii",
  "dynCall_iiiiiiiiiiiiii",
  "dynCall_iiiiiiiiiiiiiiiii",
  "dynCall_iiiiiiiij",
  "dynCall_iiiiiiifi",
  "dynCall_iiiiiiji",
  "dynCall_iiiiij",
  "dynCall_iiiiifiiii",
  "dynCall_iiiiiffi",
  "dynCall_iiiiiffii",
  "dynCall_iiiiiffiifiii",
  "dynCall_iiiiifffiiii",
  "dynCall_iiiif",
  "dynCall_iiiifi",
  "dynCall_iiij",
  "dynCall_iiijj",
  "dynCall_iiiffiiiiiii",
  "dynCall_iiiffffiii",
  "dynCall_iiidi",
  "dynCall_iij",
  "dynCall_iif",
  "dynCall_jii",
  "dynCall_jiij",
  "dynCall_fii",
  "dynCall_fiii",
  "dynCall_fiiii",
  "dynCall_fiiiii",
  "dynCall_fiiifiif",
  "ccall",
  "cwrap",
  "FS",
  "run",
  "asm",
  "calledRun",
  "stdin",
  "stdout",
  "stderr",
  "openDocument",
  "freeDocument",
  "documentTitle",
  "countPages",
  "pageWidth",
  "pageHeight",
  "pageLinks",
  "drawPageAsPNG",
  "drawPageAsHTML",
  "drawPageAsSVG",
  "loadOutline",
  "freeOutline",
  "outlineTitle",
  "outlinePage",
  "outlineDown",
  "outlineNext",
  "getPageText",
  "searchPageText",
]
`;
